constraint/pkg/client/drivers/local/local.go

```diff
@@ -0,0 +1,208 @@
+package local
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/storage"
+	"github.com/open-policy-agent/opa/storage/inmem"
+)
+
+var _ drivers.Driver = &driver{}
+
+type driver struct {
+	modulesMux sync.RWMutex
+	compiler   *ast.Compiler
+	modules    map[string]*ast.Module
+	storage    storage.Store
+}
+
+func (d *driver) Init(ctx context.Context) error {
+	return nil
+}
+
+func copyModules(modules map[string]*ast.Module, filter string) map[string]*ast.Module {
+	m := make(map[string]*ast.Module, len(modules))
+	for k, v := range modules {
+		if filter != "" && k == filter {
+			continue
+		}
+		m[k] = v
+	}
+	return m
+}
+
+func (d *driver) PutRule(ctx context.Context, name string, src string) error {
```

Comment:
Nit: unrelated to the main content of this PR, but perhaps this should be called PutModule instead?

Reply:
I like it! Changed.